packages/core/utils/src/event-bus/__tests__/build-event-names-from-entity-name.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { buildEventNamesFromEntityName } from "../../../dist"
```

Comment:
this is importing from dist, not sure if this could cause problems somewhere

Reply:
It should not be dist, i missed it 👍

Reply:
nice catch fixed it